src/lib/cms-content.ts

```diff
@@ -37,7 +37,7 @@ const itemMap = ({
   return {
     id,
     title,
-    contentType,
+    contentType: contentType === 'CaseStudy' ? 'Case Study' : contentType,
```

Comment:
In the CMS, we can't have spaces in enum fields, so this comes from the api as `CaseStudy` now.